### PR TITLE
🤖 Add start/stop buttons to the calendar view on Android

### LINKED ISSUE #7681: Add start/stop button to calendar view
## Description
We want users to be able to start and stop time entries from the calendar view. For that we need to add the start and stop button there too.

## Definition of done
- [ ] The start/stop button is added to the calendar view
- [ ] Pressing the start button opens the start view model with the default mode (running or manual)
- [ ] Long pressing the button opens the start view model with the other mode
- [ ] Pressing the stop button stops the running time entry

### PR DESCRIPTION
**Pull request by juanlaube**
## What's this?
This adds the start and stop buttons to the calendar view on Android.

### Relationships

Refs #7681 

## Why do we want this?
We want to start time entries from the calendar view too!

## How is it done?
Adding the buttons to the UI, the view model part was already done.

## :squid: Permissions
🦑

### CODE REVIEW

File: Toggl.Droid/Resources/layout/CalendarFragment.axml [hunk 1]
```diff
@@ -21,4 +21,28 @@
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintLeft_toLeftOf="parent"
         app:layout_constraintRight_toRightOf="parent" />
+    <toggl.droid.views.AnimatedFloatingActionButton
```
Comment (pdrj) [inline Toggl.Droid/Resources/layout/CalendarFragment.axml:24]: Even though you use the animated fab, you don't play the animation on long press. You can have a look at main fragment to see how it's done. 

While you're at it, would you mind adjusting the dimensions/padding on the button so the animated outline is not cutoff (if that doesn't take too much time)?

File: Toggl.Droid/Fragments/CalendarFragment.cs [hunk 2]
```diff
@@ -149,10 +150,26 @@ void swipeOnTopWeekDatePicker()
...
+            playButton.Rx()
+                .BindAction(ViewModel.StartTimeEntry, _ => true)
+                .DisposedBy(DisposeBag);
+
+            playButton.Rx()
+                .BindAction(ViewModel.StartTimeEntry, _ => false, ButtonEventType.LongPress)
+                .DisposedBy(DisposeBag);
+
+            stopButton.Rx()
+                .BindAction(ViewModel.StopTimeEntry)
```
Comment (pdrj) [inline Toggl.Droid/Fragments/CalendarFragment.cs:171]: This is related to the viewmodel - there's no call to `OnboardingStorage.StopButtonWasTapped();`

File: Toggl.Droid/Resources/layout/CalendarFragment.axml (hunk 1 above)
Comment (juanlaube) [reply Toggl.Droid/Resources/layout/CalendarFragment.axml:24]: I will have another PR where I implement the running time entry card and fix all these problems 😄

File: Toggl.Droid/Fragments/CalendarFragment.cs (hunk 2 above)
Comment (juanlaube) [reply Toggl.Droid/Fragments/CalendarFragment.cs:171]: I don't know if we should do that since we don't have onboarding tooltips for the calendar view 🤔

**Pull request merged**